src/Monolog/Handler/TelegramBotHandler.php

```diff
@@ -167,6 +187,16 @@ protected function write(array $record): void
      * @param string $message
      */
     protected function send(string $message): void
+    {
+        $messages = $this->handleMessageLength($message);
+
+        foreach ($messages as $key => $message) {
+            $key > 0 && sleep(1);
```

Comment:
sleeping inside a logger looks like a bad default behavior to me.

Reply:
Is it better to remove the delay in the hope that throttling will not have time to work (at least 5-10 messages will pass)? Or change the default behavior to truncate message?

Reply:
Fixed: changed the default behavior to truncate long message; for a split message delay can be enabled with a additional parameter